streampark-plugin/streampark-jvm-profiler/src/main/java/org/apache/streampark/plugin/profiling/util/ClassAndMethodMetricKey.java

```diff
@@ -51,13 +53,13 @@ public boolean equals(Object o) {
 
         ClassAndMethodMetricKey that = (ClassAndMethodMetricKey) o;
 
-        if (className != null ? !className.equals(that.className) : that.className != null) {
+        if (!Objects.equals(className, that.className)) {
             return false;
         }
-        if (methodName != null ? !methodName.equals(that.methodName) : that.methodName != null) {
+        if (!Objects.equals(methodName, that.methodName)) {
             return false;
         }
-        return metricName != null ? metricName.equals(that.metricName) : that.metricName == null;
+        return Objects.equals(metricName, that.metricName);
```

Comment:
` return Objects.equals(className, that.className) && Objects.equals(methodName, that.methodName) && Objects.equals(metricName, that.metricName);`